spec/02-integration/05-proxy/22-reports_spec.lua

```diff
@@ -402,39 +401,5 @@ for _, strategy in helpers.each_strategy() do
       assert.match("tcp_streams=1", reports_data[1])
       assert.match("tls_streams=0", reports_data[1])
     end)
-
-    it("#stream reports tls streams", function()
```

Comment:
This doesn't seem service mesh related. Can we still report on streams?

Reply:
I think TLS stream uses the `starttls` feature which was buggy and removed with the patches.

Reply:
In that case, shouldn't this be a `pending` instead of a delete (like the TLS-related test in `spec/02-integration/05-proxy/20-serviceless_proxying_spec.lua`)?

Reply:
Isn't this standard `nginx`:
https://github.com/Kong/kong/blob/master/kong/reports.lua#L220
http://nginx.org/en/docs/stream/ngx_stream_ssl_preread_module.html#variables

Reply:
agree with @kikito there, at the very least make this a `pending`

Reply:
I added the spec back as a `pending`